MovieQuiz/Services/StatisticService.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  StatisticService.swift
+//  MovieQuiz
+//
+//  Created by Богдан Полыгалов on 14.11.2022.
+//
+
+import Foundation
+
+final class StatisticServiceImplementation: StatisticService {
+
+    private let userDefaults = UserDefaults.standard //хранилище данных
+    
+    private enum Keys: String { //ключи для хранилища
+        case totalAccuracy, //cредняя точность (сумма всех результатов в виде - правильныеОтветы/общееКоличествоСыгранныхВопросов)
+             bestGame, //лучшая игра: результат и дата
+             gamesCount //общее количество игр
```

Comment:
Так тоже можно, имхо, мне нравится когда все кейсы на своих строчках. Плюс, еще можно использовать doc-коммент, чтобы при наведении на ключ с зажатым cmd (option) Xcode мог подсказать что это за кейс 
```suggestion
        /// cредняя точность (сумма всех результатов в виде - правильныеОтветы/общееКоличествоСыгранныхВопросов)
        case totalAccuracy 
        /// лучшая игра: результат и дата
        case bestGame 
         /// общее количество игр
        case gamesCount
```

Reply:
Дак вот для чего такие комментарии! Не знал, а когда гуглил, что-то непонятное выдавало или неслишком информативное, спасибо)